### DISCUSSION TITLE
Interfacing a power series keypad with a classic system

### DISCUSSION THREAD

**Discussion by GGagnon** [Dilbert66/esphome-dsckeybus#91] [Q&A]
Hi,
I see that I can read the power series keypad ans talk to a classic panel but can I 
generate an arduino sketch to interface a DSK PK5516ENG keypad to a PC2500 control panel?
Thanks
Gaston

**Comment by Dilbert66** [comment]
Sorry my version does not support classic panels. You can try on the original project here:  https://github.com/taligentx/dscKeybusInterface

**Reply by GGagnon** [reply]
Sorry,
Thank you for your fast answer. That was what I intended to do. sorry again.
Gaston
